app/src/main/java/de/danoeh/antennapod/fragment/NavDrawerFragment.java

```diff
@@ -354,14 +358,19 @@ public int getFeedCounterSum() {
     };
 
     private void loadData() {
+        progressBar.setVisibility(View.VISIBLE);
         disposable = Observable.fromCallable(DBReader::getNavDrawerData)
                 .subscribeOn(Schedulers.io())
                 .observeOn(AndroidSchedulers.mainThread())
                 .subscribe(result -> {
                     navDrawerData = result;
                     updateSelection(); // Selected item might be a feed
                     navAdapter.notifyDataSetChanged();
-                }, error -> Log.e(TAG, Log.getStackTraceString(error)));
+                    progressBar.setVisibility(View.GONE);
+                }, error -> {
+                    Log.e(TAG, Log.getStackTraceString(error));
```

Comment:
It's a bit strange but apparently, the code style checker is happy when doing something like this:

```suggestion
                    progressBar.setVisibility(View.GONE); },
                error -> {
                    Log.e(TAG, Log.getStackTraceString(error));
```